index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1"> 
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+    <nav id="navbar">
+      <header>HTML&CSS Documentation</header>
+     <ul>
+       <li><a href="#HTML_Introduction" class="nav-link">HTML Introduction</a></li>  
+       <li><a href="#CSS_Introduction" class="nav-link">CSS Introduction</a></li>  
+       <li><a href="#HTML_document_type_declaration" class="nav-link">HTML document type declaration</a></li>  
+       <li><a href="#HTML_Application" class="nav-link">HTML Application</a></li>  
+       <li><a href="#CSS_Advantages" class="nav-link">CSS Advantages</a></li>  
+    </ul> 
+    </nav>
+
+    <main id="main-doc">
+      
+      <section class="main-section" id="HTML_Introduction">
+        <header>HTML Introduction</header>
```

Comment:
[Required] Again for better HTML semantic, it would be better if we add subtitles as h2-h6 tags